tachyon/math/finite_fields/prime_field_fallback.h

```diff
@@ -96,7 +95,7 @@ class PrimeField<_Config, std::enable_if_t<!_Config::kIsSpecialPrime &&
     return PrimeField(big_int);
   }
 
-  constexpr static PrimeField FromMontgomery(const MontgomeryTy& mont) {
+  constexpr static PrimeField FromMontgomery(const BigInt<N>& mont) {
```

Comment:
Why not use `BigIntTy`?

Reply:
`BigIntTy` is the type to be used to use the type outside of this PrimeField. Although using `BigIntTy` also makes sense, the rest of the code here uses `BigInt<N>`.